benchmarks/asv.conf.json

```diff
@@ -81,7 +81,7 @@
         "zarr": [""],
         "pytest": [""],
         "scanpy": [""],
-        "igraph": [""],
+        "igraph-python": [""],
```

Comment:
Below, this is written `python-igraph`. Is that important?

Reply:
turns out yes, you seem to have commented on an old commit where I haven’t fixed that yet.

`python-igraph` is correct. `python-igraph` also used to be the old PyPI name which is now deprecated.